library/src/main/java/com/pokegoapi/main/CommonRequest.java

```diff
@@ -104,4 +104,24 @@ public static GetInventoryMessage getDefaultGetInventoryMessage(PokemonGo api) {
 				CommonRequest.getDownloadSettingsMessageRequest(api));
 		return serverRequests;
 	}
+
+	/**
+	 * List of common requests
+	 *
+	 * @param api The current instance of PokemonGO
+	 * @return an array of ServerRequest
+	 *
+	 */
+	public static ServerRequest[] commonRequests(PokemonGo api) {
+		ServerRequest[] serverRequests = new ServerRequest[4];
```

Comment:
you can also init the array with the values = { 
new ServerRequest(RequestType.GET_HATCHED_EGGS,
 +				GetHatchedEggsMessage.getDefaultInstance()),....
 }